### PR TITLE
Updated test.asm

### LINKED ISSUE #4: Better test.asm file
I want to patch a new test.asm file which will use most of the instructions.

### PR DESCRIPTION
**Pull request by Monal5031**
Fixed #4

### CODE REVIEW

**Comment by Monal5031** [comment]
@aviaryan Please review

**Comment by aviaryan** [comment]
@Monal5031 Please attach a screenshot of how the highlighting looks now.

**Comment by Monal5031** [comment]
@aviaryan Sir highlighting won't be affected as I just had added the autocomplete the highlighting of all the instructions was already done. So, I have shown the autocomplete in the screenshot.

File: README.md [hunk 1]
```diff
@@ -23,4 +23,4 @@ A Package to provide syntax highlighting for [LC3](http://people.cs.georgetown.e
 
 ### Screenshot
 
-![Syntax highlighting](http://i.imgur.com/a6Qufk5.png)
+![Syntax highlighting](http://imgur.com/a/GnXGK)
```
Comment (aviaryan) [inline README.md]: Not preview-able. Use a direct link to image.

**Comment by Monal5031** [comment]
@aviaryan Finally got the screenshot to have direct view.

**Comment by aviaryan** [comment]
@Monal5031 I think we should keep the previous screenshot also as it shows the GoTo feature. But this is needed only if it isn't possible to show both GoTo and autocomplete feature in one screenshot.

**Comment by Monal5031** [comment]
@aviaryan Yes, it is possible to show both these things at a time, will update the screenshot.

**Comment by aviaryan** [comment]
Thanks @Monal5031 .

**Comment by Monal5031** [comment]
Welcome @aviaryan sir.
I hope once I install my ubuntu I will look for building these kind of files from command line.

**Pull request merged**